docs/howto/operate/setup-grafana.md

```diff
@@ -0,0 +1,83 @@
+# Setting up Grafana Dashboards for a cluster
+
+This guide will walk through the steps required to setup a suite of Grafana dashboards for a cluster.
+
+## Before you start
+
+Make sure you are authenticated against the correct cloud platform and have selected the correct project to deploy to.
```

Comment:
`deploy-support` should automatically do this for you - https://github.com/2i2c-org/pilot-hubs/blob/a6f2e354399cc08275c16f49b0d92f75e11e6030/deployer/__main__.py#L46. It uses the credentials in `secrets/` and sets project info accordingly.

Reply:
Ah ok. I tend to stick with double-checking my auth before I do anything anyway, just in case! 😆